guides/common/modules/proc_adding-custom-rpm-repositories-for-centos-stream-9.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Adding_Custom_RPM_Repositories_for_CentOS_Stream_9_{context}"]
+= Adding {customrpmtitle} Repositories for CentOS Stream 9
+
+This example creates a product and repositories for CentOS Stream 9.
+
+.Prerequisite
+* Download and import the https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-rockyofficial[GPG public key] for yum repositories for Rocky Linux 9 into {Project}.
+For more information, see xref:Importing_a_Custom_GPG_Key_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content > Products*.
+. Click *Create Product* to create a product named `CentOS Stream 9`.
+For more information, see xref:Creating_a_Custom_Product_{context}[].
+. On the *Repositories* tab, click *New Repository* to create three repositories of type `yum` as follows:
++
+* *CentOS Stream 9 BaseOS*
+** *Upstream URL*: `https://mirrors.centos.org/metalink?repo=centos-baseos-$stream&arch=$basearch&protocol=https,http`
+* *CentOS Stream 9 AppStream*
+** *Upstream URL*: `https://mirrors.centos.org/metalink?repo=centos-appstream-$stream&arch=$basearch&protocol=https,http`
++
+For more information, see xref:Adding_Custom_RPM_Repositories_{context}[].
+. Click *Create Product* to create a product named `CentOS Stream 9 client`.
+. On the *Repositories* tab, click *New Repository* to create a repository of type `yum` as follows:
++
+* **CentOS Stream 9 client**
+ifndef::orcharhino[]
+** *Upstream URL*: `http://yum.theforeman.org/client/{ProjectVersion}/el9/`
```

Comment:
I thought we had a macro for this. There is at least `project-client-name`. Interestingly enough, we don't use this in `project-client-RHEL7-url` and `project-client-RHEL8-url`.

So at least this simplifies things:
```suggestion
** *Upstream URL*: `{project-client-name}el9/`
```

Reply:
Applied (even though I think the macro is named poorly) and resolves to garbage to orcharhino and Satellite; but that doesn't matter here because it will not be shown for orcharhino and Satellite.

Reply:
> even though I think the macro is named poorly

No question about it.